GPU/GLES/GPU_GLES.cpp

```diff
@@ -831,7 +831,8 @@ void GPU_GLES::CopyDisplayToOutputInternal() {
 #endif
 #endif
 
-	gstate_c.textureChanged = TEXCHANGE_UPDATED;
+	// Wait, why?
```

Comment:
You added it next to a DIRTY_ALL, which is now gone.  Probably not needed.

https://github.com/hrydgard/ppsspp/commit/ee32db47b9065004a0d70da607453ee3cd10cfb0#diff-623ed57b1a2b5e633abc1283464a6dadR131

-[Unknown]